pdns/pdnsutil.cc

```diff
@@ -266,7 +266,8 @@ int checkZone(DNSSECKeeper &dk, UeberBackend &B, const DNSName& zone, const vect
 
 
   bool hasNsAtApex = false;
-  set<DNSName> tlsas, cnames, noncnames, glue, checkglue;
+  set<DNSName> tlsas, cnames, noncnames, glue, checkglue, nsrecords;
```

Comment:
maybe rename `nsrecords` to `delegations` or `children` to make it clear the apex NSset is not included here?